internal/handlers/common/aggregations/operators/operators.go

```diff
@@ -59,12 +66,38 @@ type Operator interface {
 // The document should look like: `{<$operator>: <operator-value>}`.
 func NewOperator(doc any) (Operator, error) {
 	operatorDoc, ok := doc.(*types.Document)
-
-	switch {
-	case !ok:
+	if !ok {
 		return nil, ErrWrongType
-	case operatorDoc.Len() == 0:
+	}
```

Comment:
Could you add todo to cover the test please? `// TODO: https://github.com/FerretDB/FerretDB/pull/2789`